src/components/templates/AvatarGrid/AvatarGrid.tsx

```diff
@@ -0,0 +1,324 @@
+import UserProfilePicture from "components/molecules/UserProfilePicture";
+import firebase from "firebase/app";
+import { useSelector } from "hooks/useSelector";
+import { useUser } from "hooks/useUser";
+import React, { useCallback, useEffect, useState } from "react";
+import { WithId } from "utils/id";
+import { User } from "types/User";
+import "./AvatarGrid.scss";
+import UserProfileModal from "components/organisms/UserProfileModal";
+import { AvatarGridRoom } from "types/AvatarGrid";
+import Announcement from "./Announcement";
+import { RoomModal } from "./RoomModal";
+import ChatDrawer from "components/organisms/ChatDrawer";
+
+type Props = {
+  venueName: string;
+};
+
+const DEFAULT_COLUMNS = 40;
+const DEFAULT_ROWS = 25;
+
+const AvatarGrid = ({ venueName }: Props) => {
+  const [selectedUserProfile, setSelectedUserProfile] = useState<
+    WithId<User>
+  >();
+
+  const venueId = venueName;
```

Comment:
I suspect this is problematic, as we normally convert the venue ID. You can use `const venueId = useVenueId()` instead though.